packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -1199,6 +1199,64 @@ - (void)setZoomLevel:(CGFloat)zoom withCompletion:(void (^)(FlutterError *_Nulla
   completion(nil);
 }
 
+- (void)setVideoStabilizationMode:(FCPPlatformVideoStabilizationMode)mode
+                   withCompletion:(void (^)(FlutterError *_Nullable))completion {
+  AVCaptureVideoStabilizationMode avMode = getAvCaptureVideoStabilizationMode(mode);
```

Comment:
nit: `stabilizationMode`

Reply:
Improved, assuming you were referring to the variable avMode, therefore, avMode => stabilizationMode.